backend/census_historical_migration/make_submission.py

```diff
@@ -0,0 +1,44 @@
+from .workbooklib.end_to_end_core import run_end_to_end
+from .models import ELECAUDITHEADER
+
+from django.contrib.auth import get_user_model
+
+
+User = get_user_model()
+
+
+def load_historic_data(audit_year, dbkey):
+    """Create a submission for the given audit year and dbkey"""
+    result = {"success": [], "errors": []}
+    gen = ELECAUDITHEADER.objects.get(AUDITYEAR=audit_year, DBKEY=dbkey)
+    user = create_or_get_user(result, gen)
+    run_end_to_end(user.email, dbkey, audit_year)
+
+    return result
+
+
+def create_or_get_user(result, gen):
+    """Returns the user for the submission if found, or a default user otherwise"""
+    user_email = gen.AUDITEEEMAIL
+    user_name = gen.AUDITEECONTACT.split()[0] + "_generated"
+
+    if not user_email or len(user_email) == 0:
+        user_email = "loader_generated_email@history_data.org"
```

Comment:
For the future, this needs to be an impossible address. Something `@fac.gsa.gov` should be safe from being able to be claimed via `login.gov`.

Reply:
Done!